packages/apollo-link-error/src/__tests__/index.ts

```diff
@@ -1,7 +1,7 @@
 import gql from 'graphql-tag';
 import { ApolloLink, execute, Observable } from 'apollo-link';
-
 import { onError, ErrorLink } from '../';
+import {} from 'jest';
```

Comment:
I'm not sure what this import provides. It seems like something that should be caught with the right tsconfig. If we can remove it, then let's do it, otherwise we can leave a comment about why it's necessary.

Reply:
Oh yeah totally! Im not sure if it is something specific to webstorm, but I had to add it otherwise there are a whole bunch of tslint. I wanted to remove it before committing! removing..